src/PantheonTerminalWindow.vala

```diff
@@ -1014,6 +1020,20 @@ namespace PantheonTerminal {
             }
         }
 
+        void action_open_in_browser () {
+            if (current_terminal.uri != null) {
```

Comment:
Do you mean `if (current_terminal != null) {`? Otherwise this is redundant since you check for null uri later.